docs/architecture/adr/ADR046-replace-harbor-ecr.md

```diff
@@ -0,0 +1,43 @@
+# ADR046: Replacing harbor with ECR
+
+## Status
+
+Pending
```

Comment:
```suggestion
Accepted
```

`Pending` is a weird status... If I hit merge now, do we raise another PR just to change it with accepted? :trollface: